pokemongo_bot/cell_workers/catch_visible_pokemon.py

```diff
@@ -41,7 +41,21 @@ def work(self):
             self.bot.cell['wild_pokemons'].sort(
                 key=
                 lambda x: distance(self.bot.position[0], self.bot.position[1], x['latitude'], x['longitude']))
-            return self.catch_pokemon(self.bot.cell['wild_pokemons'].pop(0))
+
+            for pokemon in self.bot.cell['wild_pokemons']:
```

Comment:
Hmm, it doesn't look like this got split out from the other PRs. Was the intention to make this PR just the time based tasks?